File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    let list=sizes.map(item=>item.sort((a,b)=>a-b))
    
    let width=sizes[0][0]
    let height=sizes[0][1]
    for(let i=1; i<sizes.length; i++){
        width=Math.max(width, sizes[i][0])
        height=Math.max(height, sizes[i][1])
    }
    return width*height
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int width=0; // 가로가 더 길다고 가정
        int height=0;
        for(int i=0; i<sizes.length; i++){
            if(sizes[i][0]>sizes[i][1]){
                width=Math.max(width, sizes[i][0]);
                height=Math.max(height, sizes[i][1]);
            }else{
                width=Math.max(width, sizes[i][1]);
                height=Math.max(height, sizes[i][0]);
            }
        }
        return width*height;
    }
}